commitizen/commands/commit.py

```diff
@@ -72,9 +75,27 @@ def prompt_commit_questions(self) -> str:
 
         return message
 
+    def force_edit(self, message: str) -> str:
+        editor = git.get_core_editor()
+        if editor is None:
+            raise RuntimeError("No 'editor' value given and no default available.")
+        exec_path = shutil.which(editor)
```

Comment:
I think it should also be covered according to [the document of shutil](https://docs.python.org/3/library/shutil.html#shutil.which), so if users configure git editor correctly like [this](https://stackoverflow.com/questions/10564/how-can-i-set-up-an-editor-to-work-with-git-on-windows), he/she should be able to run successfully. However, this is just my assumption, I didn't try that since I don't have Windows machine...